src/EditorFeatures/Test2/ReferenceHighlighting/AbstractReferenceHighlightingTests.vb

```diff
@@ -19,16 +20,16 @@ Imports Roslyn.Utilities
 Namespace Microsoft.CodeAnalysis.Editor.UnitTests.ReferenceHighlighting
     <[UseExportProvider]>
     Public MustInherit Class AbstractReferenceHighlightingTests
-        Protected Async Function VerifyHighlightsAsync(test As XElement, Optional optionIsEnabled As Boolean = True) As Tasks.Task
-            Await VerifyHighlightsAsync(test, optionIsEnabled, outOfProcess:=False)
-            Await VerifyHighlightsAsync(test, optionIsEnabled, outOfProcess:=True)
+        Protected Async Function VerifyHighlightsAsync(test As XElement, Optional optionIsEnabled As Boolean = True) As Task
+            Await VerifyHighlightsAsync(test, optionIsEnabled, TestHost.InProcess)
+            Await VerifyHighlightsAsync(test, optionIsEnabled, TestHost.OutOfProcess)
         End Function
 
-        Private Async Function VerifyHighlightsAsync(test As XElement, optionIsEnabled As Boolean, outOfProcess As Boolean) As Tasks.Task
+        Private Async Function VerifyHighlightsAsync(test As XElement, optionIsEnabled As Boolean, testHost As TestHost) As Task
             Using workspace = TestWorkspace.Create(test)
                 WpfTestRunner.RequireWpfFact($"{NameOf(AbstractReferenceHighlightingTests)}.{NameOf(Me.VerifyHighlightsAsync)} creates asynchronous taggers")
                 workspace.TryApplyChanges(workspace.CurrentSolution.WithOptions(workspace.Options _
-                    .WithChangedOption(RemoteHostOptions.RemoteHostTest, outOfProcess)))
+                    .WithChangedOption(RemoteHostOptions.RemoteHostTest, testHost = TestHost.OutOfProcess)))
```

Comment:
Should the option value be changed to `TestHost` enum type instead of a bool?

Reply:
I'm actually removing the option in future PR.